### PR TITLE
maybe should check the result of filepath.Glob() and return an error

### PR DESCRIPTION
**Pull request by LooJee**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Description:
filepath.Glob() won't return an error while files don't exist, maybe here should have an error. Because the expandFiles() will return an empty []*File, and the error is nil. Then the yaml.Files at cmd/runner.go/Run() will be an empty slice, then the statsMgr never receive a done signal.

### CODE REVIEW

**Comment by LooJee** [comment]
I've seen the refactor pr, maybe I should close mine?

File: pkg/config/config.go [hunk 1]
```diff
@@ -408,7 +406,10 @@ func (f *File) expandFiles(dir string) ([]*File, error) {
 		}
 
 		fileNames, err := filepath.Glob(*f.Path)
-		if err != nil || len(fileNames) == 0 {
+		if len(fileNames) == 0 && err == nil {
+			err = fmt.Errorf("file: %s doesn't exist", *f.Path)
+		}
+		if err != nil {
```
Comment (veezhang) [inline pkg/config/config.go:412]: How about?
```golang
      if err != nil {
				return err
			}
			if len(fileNames) == 0 {
				return &os.PathError{Op: "open", Path: *f.Path, Err: fs.ErrNotExist}
			}
```

Comment (LooJee) [reply pkg/config/config.go:412]: it will be much better, I will resolve it promptly.

Comment (LooJee) [reply pkg/config/config.go]: @veezhang I have improved it， please check it again.

Comment (veezhang) [reply pkg/config/config.go]: @LooJee Good job!
Sorry, I find the ci is use go1.13, so maybe you should use `os.ErrNotExist` instead of `fs.ErrNotExist`.

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**